src/styles/examples.scss

```diff
@@ -138,6 +143,13 @@ article {
   transform: translate3d(0, 0, 0);
 }
 
+.button {
```

Comment:
maybe use the button style that other examples use? see showcase-components or the button style on the stream plot